stack/stack.go

```diff
@@ -0,0 +1,252 @@
+package stack
+
+import (
+	"fmt"
+)
+
+// defaultStacks contains the default stacks for given project types.
+//
+// Deprecated: please use StackOptionsMap instead.
+var defaultStacks = map[string]string{
+	"xamarin":      "osx-vs4mac-stable",
+	"cordova":      "osx-vs4mac-stable",
+	"react-native": "osx-vs4mac-stable",
+	"ionic":        "osx-vs4mac-stable",
+	"flutter":      "osx-vs4mac-stable",
+	"android":      "linux-docker-android",
+	"macos":        "osx-xcode-10.0.x",
+	"ios":          "osx-xcode-10.0.x",
+}
+
+// optionsStacks is the array of all available stacks.
+//
+// Deprecated: please use the Stacks member defined in this class.
+var optionsStacks = []string{
+	"linux-docker-android-lts",
+	"linux-docker-android",
+	"osx-vs4mac-beta",
+	"osx-vs4mac-previous-stable",
+	"osx-vs4mac-stable",
+	"osx-xamarin-stable",
+	"osx-xcode-11.0.x",
+	"osx-xcode-11.1.x",
+	"osx-xcode-11.2.x",
+	"osx-xcode-10.0.x",
+	"osx-xcode-10.1.x",
+	"osx-xcode-10.2.x",
+	"osx-xcode-8.3.x",
+	"osx-xcode-9.4.x",
+	"osx-xcode-edge",
+}
+
+// StackOptionsMap holds the available stack for the given platforms, where the first item is the default.
```

Comment:
Should maintenance check for stack change check this struct? Comment left related to the PR: https://github.com/bitrise-io/bitrise-init/pull/174/files

Reply:
I guess it should. I added that ticket as a blocker.